chalice/app.py

```diff
@@ -0,0 +1,128 @@
+import json
+import traceback
+
+from chalice import Chalice, NotFoundError, BadRequestError
+from hca.util import SwaggerAPIException
+from cloud_blobstore import BlobNotFoundError, BlobStoreUnknownError
```

Comment:
Sure, but here's the implementation of `BlobNotFoundError`:

```python
class BlobStoreError(Exception):
    pass

class BlobNotFoundError(BlobStoreError):
    pass
```

Reply:
Okay. I did not notice this before. But even though it is just a subclass of `Exception`, I still have to import it right? Otherwise, I cannot know that the exception is caused by not finding a key in the s3 bucket by just catching a broad `Exception`. Let me know if you have some better ideas for handling it.

Reply:
Well, you could just stick your own `Exception` subclasses in `__init__` (or wherever makes sense). You don't need to pull in a dependency for for that.

That said, I hadn't noticed that you're using `cloud_blobstore.s3.s3_blob_store`, which is more than just a bare `Exception` subclass, so it's probably fine.

Reply:
Okay. Cool.